src/Compilers/VisualBasic/Test/Semantic/Semantics/ScriptSemanticsTests.vb

```diff
@@ -127,7 +113,7 @@ BC2014: the value 'Nothing' is invalid for option 'ScriptClassName'
                 )
         End Sub
 
-        <Fact>
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/78792")>
```

Comment:
It would be preferable to assert what exception is thrown, if possible, rather than disable the test. If several different calls would throw exceptions, I would be fine with wrapping them all in one `Assert.Throws(() => { ... });` call.